ext/jruby/bcrypt_jruby/BCrypt.java

```diff
@@ -747,6 +758,20 @@ public static String gensalt() {
 	 * @return	true if the passwords match, false otherwise
 	 */
 	public static boolean checkpw(String plaintext, String hashed) {
```

Comment:
@Oscil8, does the below change end up fixing #82? Haven't had a chance to test yet, but if so, awesome...